### DISCUSSION TITLE
[question]: Filter empty flows?

### DISCUSSION THREAD

**Discussion by bbuchalter** [kurkle/chartjs-chart-sankey#163] [Q&A]
I am generating the data for my flows dynamically. Adding guards to ensure that each flow does not have a zero value would be a burden and feels like a reasonable option to support. Does this support exist?

**Comment by kurkle** [comment]
It is better to preprocess the data, because chart might need to do the filtering on each update. I did not take a look though, and its always possible to cache things - at the cost of complexity.
